docs/config.rst

```diff
@@ -1,49 +1,35 @@
 Globus SDK Configuration
 ========================
 
-There are three standard, canonical locations from which the Globus SDK will
```

Comment:
Comparing the old and new versions of this doc, I think there's no question that the new system is easier for users to understand, in addition to being easier to maintain.
But also, I'm not sure why we bothered defining INI for people... I guess someone was confused at some point. Good riddance to this section.